Added *.db files to stdApp/Db/Makefile so the standard db/ folder wou…

oksanagit opened the pull request:
Hi, 
Though I needed only PIDrecord related files, I added them all. The files are grouped what I considered logical based on their names.
Thanks,
Oksana

oksanagit committed:
Added *.db files to stdApp/Db/Makefile so the standard db/ folder would be created and these files will be copied there

MarkRivers commented:
I like the idea of installing database files, and I think it should be done in all synApps repositories.

It is easier to maintain if you don't list the individual files in the Makefile, but rather use wildcards.  

These commands should install all .template and .db files.

```
DB += $(patsubst ../%, %, $(wildcard ../*.template))
DB += $(patsubst ../%, %, $(wildcard ../*.db))
```
I just tried it and it seems to work OK.

ralphlange commented:
Agreed, iff all db files are for public consumption.
(Test databases and similar "private" things should probably stay private.)

Also use the opportunity to double check db file names. When using multiple directories in an IOC database search path, name clashes may not be obvious and still provide interesting surprises.

oksanagit committed:
removed explicit listing of the .db files. Added wildcard make expressions as suggested by Mark Rivers.

oksanagit commented:
Hi,
I very much support Mark's suggestion to make .db files propagated to the db/ folder in all modules. It will allow debian packages to distribute those files to the users, and areaDetector gracefully work with debian packages. I have modified Makefile as suggested by Mark, and added another line to include .vdb files.  Please take a look.
